cpp_linter/rest_api/github_api.py

```diff
@@ -522,7 +656,138 @@ def _dismiss_stale_reviews(self, url: str):
+            strict=False,
+        )
+        if response.status_code != 200:
+            logger.error(
+                "Failed to %s review thread comment: %d",
+                operation,
+                response.status_code,
+            )
+        elif "errors" in response.json():
+            error_msg = response.json()["errors"][0]["message"]
+            if "Resource not accessible by integration" in error_msg:
+                logger.error(
+                    "Changing review thread comments requires `contents: write` permission."
+                )
+            else:
+                logger.error(
+                    "Failed to %s review thread comment: %s", operation, error_msg
+                )
+        else:
+            logger.debug("Review comment thread %sd: %s", operation, thread_id)
+
+    def _hide_stale_reviews(self, ignored_reviews: List[str]):
+        """Hide all review comments that were previously created by cpp-linter
+
+        :param ignored_reviews: List of review comments to keep displayed.
+        """
+        url = f"{self.api_url}/repos/{self.repo}/pulls/{self.pull_request}/reviews"
+        next_page: Optional[str] = url + "?page=1&per_page=100"
+        while next_page:
+            response = self.api_request(url=next_page)
+            next_page = self.has_more_pages(response)
+            reviews: List[Dict[str, Any]] = response.json()
+            for review in reviews:
+                if (
+                    "body" in review
+                    and cast(str, review["body"]).startswith(COMMENT_MARKER)
+                    and review["node_id"] not in ignored_reviews
+                ):
+                    mutation = HIDE_REVIEW_COMMENT
+                    variables = {
+                        "subjectId": review["node_id"],
+                    }
+                    response = self.api_request(
+                        url=f"{self.api_url}/graphql",
+                        method="POST",
+                        data=json.dumps({"query": mutation, "variables": variables}),
+                        strict=False,
+                    )
+                    if response.status_code != 200:
+                        logger.error(
+                            "Failed to hide review comment: %d", response.status_code
+                        )
+                    elif "errors" in response.json():
+                        error_msg = response.json()["errors"][0]["message"]
+                        if "Resource not accessible by integration" in error_msg:
+                            logger.error(
+                                "Hiding review comments requires `contents: write` permission."
+                            )
+                        else:
+                            logger.error("Hiding review comment failed: %s", error_msg)
+                    else:
+                        logger.debug("Review comment minimized: %s", review["node_id"])
```

Comment:
Again, `self.api_request()` comes equipped with error logging. So, if we just want to add context to the logged error, we can just say:
```suggestion
                    logger.debug(
                        "Minimized review comment: %s (node_id: %s)",
                        repr(response.status_code != 200).lower(),
                        review["node_id"],
                    )
```